CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+- Delete API parameters from wazuh.yml template ([#1155](https://github.com/wazuh/wazuh-puppet/pull/1155)) \-
```

Comment:
Add  ` (Puppet Module)` at the end of line 13, please.

Reply:
done